src/lib/src/internals.ts

```diff
@@ -91,6 +91,12 @@ export class Spectator<C> {
    * @param selector string | Element | DebugElement
    */
   click(selector: string | Element | DebugElement) {
```

Comment:
We should have a type, instead of repeating this. 
`type SpectatorElement = string | Element | DebugElement | Window`